src/lib.rs

```diff
@@ -104,6 +104,11 @@
 //! (unless they're [subcommands](#subcommands)),
 //! and you can adjust these apps and args by `#[structopt(...)]` [attributes](#attributes).
 //!
+//! The derived `StructOpt` trait allows access to other methods besides
+//! `from_args`. You can, for instance, get the underlying `clap::ArgMatches`
+//! by doing `Opt::clap().get_matches()`. See the [trait's reference documentation](trait.StructOpt.html)
+//! for more details.
```

Comment:
You should use `[trait's reference documentation](StructOpt)` instead of explicit file name, see [the RFC](https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md)

Reply:
Can't get this to work. I've tried these two syntaxes:

```md
    [trait's reference documentation](StructOpt)
```

```md
    [trait's reference documentation](structopt::StructOpt)
```

I'm running `cargo doc` and rustdoc seems to be version 1.34.2. Should I do anything differently?

Reply:
I can't make it work either :cry: Forget about it.